### PR TITLE
Update Spring version

### PR DESCRIPTION
**Pull request by mmoayyed**
Title says all.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
 mongoJavaDriverVersion=3.4.2
 guavaVersion=21.0
 fongoVersion=2.0.12
-springVersion=4.3.6.RELEASE
+springVersion=4.3.11.RELEASE
 springBootVersion=1.4.4.RELEASE
```
Comment (bartoszwalacik) [inline gradle.properties]: 1.5.7.RELEASE ?

**Comment by bartoszwalacik** [comment]
unfortunately, it wasn't that simple https://github.com/javers/javers/pull/580/commits/5545243e3835d1fef6688599e2bf9cd437d154a6 :(

**Comment by mmoayyed** [comment]
That's a pity. What else should I do to fix the error?

**Comment by bartoszwalacik** [comment]
I've also upgraded Spring Boot to 1.5.7, and now build is faling on Travis, I can't reproduce it on local env, see https://travis-ci.org/javers/javers/builds

**Comment by bartoszwalacik** [comment]
Released in 3.5.1

**Pull request merged**